README.md

```diff
@@ -29,35 +27,17 @@ where everything falls into place in the new approach.
 | `{integration}/test_check.py` | `{integration}/tests/*.py`                        |
 | n/a                           | `{integration}/setup.py`                          |
 
-## A note about installing
-
 Now that integrations are cleanly defined as python packages, we will soon be able
 to ship them as Python wheels that will be pip-installable in the Python environment
 embedded into the Datadog Agent. This presents a paradigm change in the way we will
 be delivering standalone integration upgrades, moving away from OS-specific packages
 to idiomatic Python package delivery.
 
-Agent releases starting from version 5.21 bundle the latest wheels for any
-integration, but at the moment you can't upgrade or downgrade between releases.
-
-Each Datadog Agent release will continue to ship a set of the most up to date stable integrations available. The `requirements-agent-release.txt` file at the root of this repo is the best place to check what Integration version is shipped with each Agent.
-
-**Note** The release process is currently in flux as we move toward the ability to ship wheels independently of Agent releases. Due to this, the Changelog may show a version and release that isn't yet available to download. Please check the below table to see which Integration versions are shipped with your Agent install.
-
-| Agent Version   | List of Shipped Integration Versions   |
-| --------------- | -------------------------------------- |
-| 6.2.1           | [Link][15]                             |
-| 6.3.0           | [Link][16]                             |
-
 ## Quick Start
 
-Working with integrations is easy, the main page of the [development docs](docs/dev/README.md)
```

Comment:
now that dev docs are pulled from the docs website, we shouldn't link markdown docs anymore